### DISCUSSION TITLE
Event examples

### DISCUSSION THREAD

**Discussion by actualdankcoder** [bwmarrin/discordgo#1194] [Q&A] [answered]
Are there any examples on discord events like MESSAGE_CREATE, PRESENCE_UPDATE, etc

**Comment by 42atomys** [comment, marked as answer]
Hi, you have a very simple example about handling event (like messageCreate) on ping pong example available here https://github.com/bwmarrin/discordgo/blob/master/examples/pingpong/main.go


All handlers can be found in your IDE or on the `eventhandler.go` source file (https://github.com/bwmarrin/discordgo/blob/c8c153872c36279e63107bc3d085de8d38e6c77f/eventhandlers.go#L1294)


Best regards :)
